lcgs-examples/fischer/1/ensure-mutual-exclusion-TRUE-01.atl

```diff
@@ -0,0 +1 @@
+[[p0,p1]] G (mutual_exclusion)
```

Comment:
The LCGS does not have a label called `mutual exclusion`

Reply:
I have renamed the label in 4f94bcb